src/directives/formly-form.js

```diff
@@ -16,7 +16,7 @@ module.exports = ngModule => {
       restrict: 'E',
       template: function(el, attrs) {
         /* jshint -W033 */ // this because jshint is broken I guess...
-        var rootEl = attrs.rootEl || 'ng-form';
+        var rootEl = attrs.rootEl || 'form';
```

Comment:
Ok, would you be fine with it to only do that when running in IE8?
Or do a <div ng-form> when running in IE8?
IE8 can't cope with ng-form and directive replace because for some reason custom tags in IE8 look like having multiple root nodes.